http/org_service.go

```diff
@@ -73,6 +73,41 @@ const (
 	organizationsIDLabelsIDPath      = "/api/v2/orgs/:id/labels/:lid"
 )
 
+func checkOrganizationExists(handler OrgHandler, r *http.Request) error {
+	ctx := r.Context()
+	req, err := decodeGetOrgRequest(ctx, r)
+	if err != nil {
+		return &influxdb.Error{
+			Err: err,
+		}
+	}
+
+	_, findErr := handler.OrganizationService.FindOrganizationByID(ctx, req.OrgID)
+	if findErr != nil {
+		return &influxdb.Error{
+			Err: findErr,
+		}
+	}
+
+	return nil
+}
+
+func checkOrganziationExistsWrapper(handler *OrgHandler) func(http.HandlerFunc) http.HandlerFunc {
```

Comment:
go on and take advantage of the type you just created `func organziationExists(handler *OrgHandler) Middleware {`

one other thing here is that you are passing the org handler as a value instead of using pointer semantics. you want to stick with one or the other to be consistent here. I'd recommend using pointer semantics here as that's what we are sharing across these functional boundaries